src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -2807,19 +2883,20 @@ private void onRequestWindow(
                 if (requestBudgetId != 0L && requestDebitorIndex == NO_DEBITOR_INDEX)
                 {
                     requestDebitor = supplyDebitor.apply(budgetId);
-                    requestDebitorIndex = requestDebitor.acquire(budgetId, initialId, this::flushRequestDataIfNecessary);
+                    requestDebitorIndex = requestDebitor.acquire(budgetId, initialId, decodeNetworkIfNecessary);
```

Comment:
Perhaps we can use `Http2Server.this::decodeNetworkIfNecessary` instead of using a separate `decodeNetworkIfNecessary` member field?